src/linkediting.js

```diff
@@ -220,6 +229,68 @@ export default class LinkEditing extends Plugin {
 		} );
 	}
 
+	/**
+	 * Attaches handlers for {@link module:engine/view/document~Document#event:enter} and
+	 * {@link module:engine/view/document~Document#event:click} to enable link following.
+	 *
+	 * @private
+	 */
+	_enableLinkOpen() {
+		const editor = this.editor;
+		const view = editor.editing.view;
+		const viewDocument = view.document;
+		const modelDocument = editor.model.document;
+
+		this.listenTo( viewDocument, 'click', ( evt, data ) => {
+			const shouldOpen = env.isMac ? data.domEvent.metaKey : data.domEvent.ctrlKey;
```

Comment:
It's interesting that we can distinguish between a meta key and a ctrl key. But that's probably because I'm not a Mac user.